src/DesktopVirtualization/custom/New-AzWvdMsixpackage_PackageAlias.ps1

```diff
@@ -0,0 +1,158 @@
+function New-AzWvdMsixPackage_PackageAlias {
```

Comment:
We don't use `_` in our cmdlet names. I suggest to rename it to `New-AzWvdMsixPackageAlias`.

Reply:
The '_' is actually not part of the cmdlet name. Does it still matter? this is the official cmdlet:
        $package_created = **New-AzWvdMsixPackage** -PackageAlias 'msixpackage' `
            -ImagePath 'C:\msix\singlemsix.vhd' `
            -HostPoolName $env.HostPool `
            -ResourceGroupName $env.ResourceGroup `
            -SubscriptionId $env.SubscriptionId  `
            -DisplayName 'package-Alias-test' `
            -IsActive 

I was just following the same convention as for the custom cmdlet for New-AzWvdApplication_AppAlias